### ISSUE TITLE
Session limit has been reached for current device.

### ISSUE THREAD

**Issue by AnPang940** [Emotiv/cortex-example#77]
Operating system: win10 edu
Last upstream commit (run `git log --author="Robert Lord" | head -n 1`): 
language version:(nodejs version, ruby version ....) nodejs
Cortex version: V.1.7.1-x86

-
when I try to run the raw.js in nodejs and the error comes out lile 
JSONRPCError: Session limit has been reached for current device.

**Comment by bchaperon** [comment]
Hello @AnPang940 
This error probably is related to your license.
With some licenses, you can only create a limited number of session every month.
What kind of license do you have?

**Comment by AnPang940** [comment]
> Hello @ AnPang940
> 此错误可能与您的许可证有关。
> 对于某些许可证，您每月只能创建有限数量的会话。
> 你有什么样的执照？

**Comment by AnPang940** [comment]
hi @bchaperon 

and my ID is "beiligong",is there any way to get it clear ?

cause I need to work with it!

**Comment by AnPang940** [comment]
@bchaperon 
Thanks for your reply. Our license does have a limited number of session every month. 
But, we need to run the program of raw.js in nodejs to get the rawdata of EEG. 
So, we will creat a new seesion everytime for running the program of raw.js.
And then , we will face the problem of " Session limit has been reached for current device".
When I try to close it manually with updateSession again I get the response "Session does not exist".
How can I deal with it?

**Comment by MichaelaCF** [comment]
I still have the same problem... is there any news on this?

**Comment by nguoithichkhampha** [comment]
@MichaelaCF do you use latest version (1.8.1) ? 
if yes please try to call "authorize" with "debit"=100.

**Comment by MichaelaCF** [comment]
Hi @nguoithichkhampha, thanks for your reply! 
I could not find the version number vor CortexUI, but I tried to update it with the maintenance tool and it did not find any update, so I guess it is at the latest version. When authorizing I had debit at something very high, like 10000. (To be honest I do not fully understand what the debit parameter is doing...) 
I tried it with 100 and it is actually working! But when I change it so something higher now, it is also working... very strange. Was there an internal update at emotiv or something, because I didn't change anything at my code since yesterday, and it was not working then. 
Well, I hope it stays like this :D 
I still get the error when I try to close a session, but if the problem with "Session limit has been reached" is gone, I can absolutely live with that.
